### ISSUE TITLE
Problem: The `Navigation` doesn't look presentable between 1200px and 480px

### ISSUE THREAD

**Issue by predrag-us** [holdex/holdex-venture-studio#446]
Problem: The `Navigation` doesn't look presentable between 1200px and 480px.
Solution: Come up with a new design for the navigation ⏤ the one that works for every breakpoint. 

Example of how the `Navigation` looks now at 1200px:

![Image](https://github.com/holdex/holdex-venture-studio/assets/18451941/c789d2d1-1925-4175-9a66-77a543480c87)

--- 

### [Figma Design ⏤ Component](https://www.figma.com/file/39MGGVIGQ1bL3FgVzuXEdC/Holdex?type=design&node-id=3431%3A52392&mode=design&t=GAoqvPjrMDyymaPq-1)
**Mind the note alongside the component.**

--- 

Note: The new Navigation is currently implemented only on `/` in the design (linked preview). I'll apply the changes to all screens gradually.

**Comment by winter-sols** [comment]
@predrag-us 
This is our current implmentation
![image](https://github.com/holdex/holdex-venture-studio/assets/100481931/f5518101-2d0a-45ad-8517-a0e20984caba)
and this is the new design
![image](https://github.com/holdex/holdex-venture-studio/assets/100481931/69e43ba9-2f9e-4b17-8567-e98651d61943)
Do I need to update the header following the new design?

**Comment by winter-sols** [comment]
@predrag-us what do you mean this sentence?
"Problem: The Navigation doesn't look presentable between 1200px and 480px."
I can see the header always including screen size of between 1200px and 480px.

**Comment by winter-sols** [comment]
@predrag-us gotcha!

**Comment by markholdex** [comment]
@winter-sols this problem has a low priority. We execute on Urgent and High priority Problems first. Please double-check the priority before starting to resolve it. Thanks. 
<img width="1364" alt="Screenshot 2023-07-19 at 09 44 29" src="https://github.com/holdex/holdex-venture-studio/assets/46079535/6cc9138e-9380-43fe-88ca-923673f77057">
